data/gegno/gegno intro missions.txt

```diff
@@ -36,7 +36,7 @@ mission "First Contact: Gegno Vi"
 			branch "return encounter"
 				has "First Contact: Gegno Vi: deferred"
 			
-			`As you descend towards the surface of this seemingly uninhabited world, you active your ship's radar system to scan the surface for anything of interest. While waiting for something to appear, you look up and catch a glimpse of a cloud of dust bursting from the ground far off to your right. Peering back down to your radar, you check the surface scans of that direction, but nothing appears. As you continue your flight, another sudden dust cloud appears several minutes later, but still nothing on the radar. You decide to keep your ship a few thousand meters above the surface and take the time to carefully look around, but there still doesn't seem to be anything of note besides the cracked rock formations and valleys of sand.`
```

Comment:
Not so sure about the correction of this here and the other place. You are catching "a glimpse of a cloud of dust...," not "a glimpse of a dust bursting..."

Simplifying it to just "catch a glimpse of dust" also wouldn't work properly as you're seeing dust everywhere regardless.